CHANGELOG.md

```diff
@@ -48,6 +48,30 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 - Changed version of `github.com/swaggo/swag/cmd/swag` from v1.7.0 to v1.7.1 in
```

Comment:
A note on the glossary in the PR description:

> - test function: A single test function. = TestResultDetail
> - UT: A single test file, collection of test functions. = TestResultSummary
> - test run: A collection of UTs, multiple files possible. = SummaryOfTestResultSummaries

For one thing they are not guaranteed to be unit tests. Even TRX files can include integration tests, benchmark tests, end-to-end tests, et.al.

And a single uploaded test file/artifact with test results is more aligned with a "test run" and not a collection of files. Altough that's mostly true for our dotnet samples, but not sure if other test libs create multiple files for different subprojects, category, or whatever.

This is pretty late in the review process, so it's fully OK to ignore this suggestion, but maybe something like this:

> - test result: A single test = TestResult
> - test results file: A single test file, collection of tests. = TestResultSummary
> - build test results: A summary of test results for a given build, multiple files possible. = BuildTestResultSummary